data/transit/route/bus.json

```
@@ -3730,6 +3710,9 @@
       },
```

If these are duplicates, you should add matchNames so that existing bus routes/stops with this network value get matched with this entry:
```suggestion
      },
      "matchNames": ["Bus Éireann Dublin Commuter"],
```